javascript-if/main.js

```diff
@@ -0,0 +1,79 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  }
+
+}
+
+function isEven(number) {
+  if (number % 2 > 0) {
+    return false;
+  } else {
+    return true;
+  }
+}
+
+function startsWithJ(string) {
+  if (string.charAt(0) === 'J') {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrink(person) {
+  if (person.age < 21) {
+    return false;
+  } else {
+    return true;
+  }
+}
+
+function isOldEnoughToDrive(person) {
+  if (person.age < 16) {
+    return false;
+  } else {
+    return true;
+  }
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  if ((person.age < 18) && (person.age < 21)) {
+    return false;
+  }
+}
```

Comment:
What's this return if `person.age` is at least `21`?

Reply:
I misunderstood the function 
so I added the legal drinking age too 
I will delete now!